doc/whats_new.rst

```diff
@@ -24,8 +24,13 @@ Legend for changelogs
 - |API|: you will need to change your code to have the same effect in the
   future; or a feature will be removed in the future.
 
-.. include:: whats_new/v0.22.rst
-.. include:: whats_new/v0.21.rst
+Latest Releases
+=================
+.. toctree::
+    :maxdepth: 3
```

Comment:
Why this choice of maxdepth? I'd consider less, or even to just make this part of the same listing as below.

Reply:
This was to keep more details about recent releases wrt the older ones, and stay close to the actual version in some way. `maxdepth:3` gives you the list of the  modules affected by modifications.
If this is usefulness I will change it.